main.py

```diff
@@ -0,0 +1,51 @@
+from typing import List
+
+from fastapi import FastAPI, Depends, HTTPException
+from sqlalchemy.orm import Session
+
+import crud
+import schemas
+from database import SessionLocal
+
+app = FastAPI()
+
+
+def get_db() -> Session:
+    db = SessionLocal()
+    try:
+        yield db
+    finally:
+        db.close()
+
+
+@app.get("/authors/", response_model=List[schemas.Author])
+def get_authors(skip: int = 0, limit: int = 10, db: Session = Depends(get_db)):
+    return crud.get_authors(db=db, skip=skip, limit=limit)
+
+
+@app.post("/authors/", response_model=schemas.Author)
+def post_author(author: schemas.AuthorCreate, db: Session = Depends(get_db)):
+    return crud.create_author(db=db, author=author)
+
+
+@app.get("/authors/{author_id}/", response_model=schemas.Author)
+def get_author(author_id: int, db: Session = Depends(get_db)):
+    db_author = crud.get_authors_by_id(db=db, author_id=author_id)
```

Comment:
Ensure that `crud.get_authors_by_id` correctly retrieves an author by ID. This function should return `None` if the author is not found, which is handled by the subsequent `HTTPException`.